vm/api.cpp

```diff
@@ -287,7 +287,7 @@ SourcePawnEngine2::LoadBinaryFromFile(const char* file, char* error, size_t maxl
     }
   }
 
-  if (!pRuntime->Name())
```

Comment:
This thing bugs me still, they're very different checks.

Reply:
The old one was broken by previous refactoring elsewhere, the new one is the intended behaviour.